packages/components/src/Chips/InternalChipMultiSelect.tsx

```diff
@@ -33,7 +33,18 @@ export function InternalChipMultiSelect({
             <InternalChip
               {...chip.props}
               active={isChipActive}
-              suffix={checkmarkIcon(isChipActive)}
+              {...(isChipActive
+                ? {
+                    suffix: (
+                      <Icon
+                        size="small"
+                        name="checkmark"
+                        color="interactiveSubtle"
+                      />
+                    ),
+                  }
+                : {})}
+              tabIndex={undefined}
```

Comment:
Same as https://github.com/GetJobber/atlantis/pull/1995/files#r1736369895
```suggestion
              tabIndex={-1}
```

Reply:
ah right. addressed both of these.